src/ProjectTemplates/scripts/Test-Template.ps1

```diff
@@ -23,7 +29,10 @@ function Test-Template($templateName, $templateArgs, $templateNupkg, $isSPA) {
   <Import Project=""$PSScriptRoot/../test/bin/Debug/netcoreapp3.0/TemplateTests.props"" />
   <ItemGroup>
     <PackageReference Include=""Microsoft.NET.Sdk.Razor"" Version=""`$(MicrosoftNETSdkRazorPackageVersion)"" />
-  </ItemGroup>")
+  </ItemGroup>
+  <PropertyGroup>
+    <DisablePackageReferenceRestrictions>true</DisablePackageReferenceRestrictions>
```

Comment:
What is this wizardry?

FYI, I enabled something similar by moving all these logic to a target inside MSBuild in some other changes I'm making. Now you can just run a target and point it to a given folder and it will create a directory.build.props and directory.build.targets with all the things you need for this to work with `dotnet new -i templatepackage`